modules/tracer/backend/src/main/scala/command.trace.scala

```diff
@@ -107,7 +114,6 @@ def Trace(
             .through(inBytes.publish)
             .onFinalize(
               Logging.info("process stdin finished, shutting down tracer") *>
-                child.terminate *>
```

Comment:
Tests pass, but I feel like we may be missing something if this isn't called. The fs2 process API doesn't actually have a method for killing the process, guess you have to somehow provoke the resource to exit early instead.

How would I confirm whether this is fine as-is?